app/models/schedule.rb

```diff
@@ -1,10 +1,8 @@
 class Schedule < ActiveRecord::Base
-  has_and_belongs_to_many :order_cycles, join_table: 'order_cycle_schedules'
   has_paper_trail meta: { custom_data: :order_cycle_ids }
 
-  has_many :coordinators, uniq: true, through: :order_cycles
-
-  attr_accessible :name, :order_cycle_ids
+  has_and_belongs_to_many :order_cycles, join_table: 'order_cycle_schedules'
+  has_many :coordinators, -> { uniq }, through: :order_cycles
```

Comment:
what is that `uniq` here? I don't see any var defined with that name, at least in its surroundings :thinking: is it maybe a scope?

Reply:
must be this https://apidock.com/rails/ActiveRecord/QueryMethods/uniq